internal/events/dx_callbacks.go

```diff
@@ -213,11 +213,13 @@ func (em *eventManager) privateBlobReceived(dx dataexchange.Plugin, event dataex
 	// Dispatch to the blob receiver for efficient batch DB operations
 	em.blobReceiver.blobReceived(em.ctx, &blobNotification{
 		blob: &core.Blob{
+			Namespace:  br.Namespace,
 			Peer:       br.PeerID,
 			PayloadRef: br.PayloadRef,
 			Hash:       &br.Hash,
 			Size:       br.Size,
 			Created:    fftypes.Now(),
+			DataID:     fftypes.MustParseUUID(br.DataID),
```

Comment:
Bit worried about the `Must` here. Does this introduce a case where a DX plugin with a malformed `dataId` in the event notification could panic the server?

Reply:
Good point. I'll make this a bit safer.